docs/core/tutorials/cli-templates-create-project-template.md

```diff
@@ -139,7 +123,7 @@ The following template packages will be installed:
 Success: <root path>\working\templates\consoleasync installed the following templates:
 Templates                                         Short Name               Language          Tags
 --------------------------------------------      -------------------      ------------      ----------------------
-Example templates: async project                  consoleasync             [C#]              Common/Console/C#9
+Example templates: async project                  consoleasync             [C#]              Common/Console
```

Comment:
above in line 111 
extra files folders and files you don't want
-->
extra folders and files you don't want